src/constants.ts

```diff
@@ -91,36 +90,13 @@ export const QUESTIONAIRRE: inquirer.QuestionCollection = [
     }
   },
 
-  // TESTING BACKEND
-  {
-    type: 'list',
-    name: 'backend',
-    message: 'Where do you want to run your e2e tests?',
-    choices: [
-      {name: 'On my local machine', value: 'local'},
-      {name: 'On a remote machine (cloud)', value: 'remote'},
-      {name: 'Both', value: 'both'}
-    ],
-    default: 'local'
-  },
 
-  {
-    type: 'list',
-    name: 'cloudProvider',
-    message: '(Remote) Please select your cloud provider:',
-    choices: [
-      {name: 'BrowserStack', value: 'browserstack'},
-      {name: 'Sauce Labs', value: 'saucelabs'},
-      {name: 'Other providers or remote selenium-server', value: 'other'}
-    ],
-    when: (answers) => ['remote', 'both'].includes(answers.backend)
-  },
 
   // BROWSERS
   {
     type: 'checkbox',
     name: 'browsers',
-    message: 'Which browsers will you be testing on?',
+    message: 'Select target browsers',
     choices: (answers) => {
       let browsers = BROWSER_CHOICES;
       if (answers.backend === 'local' && process.platform !== 'darwin') {
```

Comment:
```suggestion
      if (process.platform !== 'darwin') {
```

Reply:
But this will also remove 'Safari' for users who want to test on Safari browser on remote/cloud service from Windows/Linux machine.

Maybe we can just remove this `if` block so that 'Safari' appears everytime, or if the user's system is non-mac, we can change the option to something like 'Safari (cloud only)'?

Reply:
Adding Safari (cloud only) will only complicate things further because we have to handle the case where you select Safari and do not select cloud. So I think for now it's ok to just show Safari for Mac because advanced users who want to test on cloud safari can always add the config manually.

Reply:
so an error will already be thrown?